packages/server-core/src/cluster/api-job/api-job.test.ts

```diff
@@ -0,0 +1,94 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import assert from 'assert'
+
+import { destroyEngine } from '@etherealengine/engine/src/ecs/classes/Engine'
+
+import { apiJobPath } from '@etherealengine/engine/src/schemas/cluster/api-job.schema'
+import { Application } from '../../../declarations'
+import { createFeathersKoaApp } from '../../createApp'
+
+const getRandomizedName = (name: string, suffix = '', prefix = 'test') =>
+  `${prefix}-${name}-${(Math.random() + 1).toString(36).substring(7)}${suffix}`
+
+/**appends trailing `/` for directory paths */
+const getDirectoryPath = (name: string) => name + '/'
+
+describe('api job service', () => {
+  let app: Application
+  before(async () => {
+    app = createFeathersKoaApp()
+    await app.setup()
+  })
+  after(() => {
+    return destroyEngine()
+  })
+
+  it('should register the service', () => {
+    const service = app.service(apiJobPath)
+    assert.ok(service, 'Registered the service')
+  })
+
+  it('find service', () => {
+    assert.doesNotThrow(async () => await app.service(apiJobPath).get(''))
+  })
+
+  let jobId
+  it('creates a job', async () => {
+    const createdJob = await app.service(apiJobPath).create({
+      name: 'test-job',
+      startTime: new Date().toISOString().slice(0, 19).replace('T', ' '),
```

Comment:
We can use `toDateTimeSql` from `packages/server-core/src/util/datetime-sql.ts` here and on next line.